doc/faq.rst

```diff
@@ -303,13 +303,3 @@ Pylint, you can set ``evaluation`` to the above expression to get the new
 behavior. Likewise, since negative values are still technically supported,
 ``evaluation`` can be set to a version of the above expression that does not
 enforce a floor of zero.
-
-6.2 I think I found a bug in Pylint. What should I do?
```

Comment:
This doesn't hurt right?

Reply:
Imo the FAQ is just something were we duplicate information because the overall organization of the doc is horrible. I'm aiming to removing it (almost entirely).